ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -401,14 +401,20 @@ private void bindAccount(PreparedStatement statement, Account account, Statement
    */
   private void bindContainer(PreparedStatement statement, int accountId, Container container,
       StatementType statementType) throws SQLException {
+    String containerInJson;
+    try {
+      containerInJson = objectMapper.writeValueAsString(container);
+    } catch (IOException e) {
+      throw new SQLException("Fail to serialize container: " + container.toString(), e);
```

Comment:
It doesn't make sense to throw a java.sql.SqlException for deserialization failure. Maybe throw IOException?

Reply:
This method only throws out an SQLException, that's why I am throwing out SQLException here even for serialization error.

Reply:
We can change the signature to include the correct exception.